docs/roadmap.md

```diff
@@ -13,7 +13,7 @@ only, and this section may be revised to provide newer information at any time.
 
```

Comment:
Maybe we should take the opportunity of this PR to move this file to doc/roadmap.md (the link to readme in main README.md need to be adapted also)

Reply:
done in https://github.com/telefonicaid/iotagent-node-lib/pull/1208/commits/9ea204271e50ffa307bbf9000f8d2e5f891d40a7